### PR TITLE
3244547 : [PPE][Light Mode]- [BYOB] Not getting any bot response for signin card

### PR DESCRIPTION
**Pull request by akhiloct1510**
### Id of the task, bug, story or other reference.
https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3244547/

### Description
The signin card (adaptive card) from BOT is not rendering in LCW

## Solution Proposed
_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

### Acceptance criteria
Return activity if the signinid is not extracted using Sign-in URL. BOT owner should be responsible of configuring the correct auth URL to continue with the functionality.

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

image

image

### Sanity Tests
-   [X] You have tested all changes in Popout mode
-   [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
-   [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y
- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

__*Please provide justification if any of the validations has been skipped.*__

### CODE REVIEW

File: chat-widget/src/components/livechatwidget/common/ActivitySubscriber/BotAuthActivitySubscriber.ts [hunk 1]
```diff
@@ -86,7 +86,10 @@ export class BotAuthActivitySubscriber implements IActivitySubscriber {
         const signInId = extractSignInId(signInUrl);
 
         if (!signInId) {
-            return;
+            TelemetryHelper.logLoadingEvent(LogLevel.INFO, {
+                Event: TelemetryEvent.BotAuthActivityUndefinedSignInId
```
Comment (charliewang95) [inline chat-widget/src/components/livechatwidget/common/ActivitySubscriber/BotAuthActivitySubscriber.ts:90]: Please update telemetry.md with this change

Comment (charliewang95) [inline chat-widget/src/components/livechatwidget/common/ActivitySubscriber/BotAuthActivitySubscriber.ts:90]: Please update changelog

Comment (akhiloct1510) [reply chat-widget/src/components/livechatwidget/common/ActivitySubscriber/BotAuthActivitySubscriber.ts:90]: Done

**Review by xTEddie — APPROVED** [review]

**Review by charliewang95 — APPROVED** [review]

File: CHANGE_LOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Fixed
+
+- Fixed an issue where sign-in card cannot be rendered
```
[the remark this answers is not in the document]
Comment (akhiloct1510) [reply CHANGE_LOG.md]: Done
